### DISCUSSION TITLE
Multiple basePdfs in one pdf

### DISCUSSION THREAD

**Discussion by MagicalBloom** [pdfme/pdfme#145] [Q&A] [answered]
Is it possible to have different basePdf for different pages in some way? In my case I'd like to have a cover page which is different from rest of the pages (these are created dynamically and amount of pages may vary) and I couldn't find a way to do this.

**Comment by hand-dot** [comment, marked as answer]
At this moment, we can't. But you can use other library as pdf merger.

I wanna avoid basePdf’s multi-case.
Does basePdf has multiple page or not?

That's why I don't implement multi basePdf

**Reply by MagicalBloom** [reply]
Alright I'll go with the merge route.
